test/app/modes/C24O6H8_extfile/modes_in.hsd

```diff
@@ -0,0 +1,19 @@
+# Needs the equilibrium geometry, at which the Hessian had been calculated
+Geometry = GenFormat {
+  <<< geom.gen
+}
+
+# You need to specify the SK-files, as the mass of the elements is needed
```

Comment:
From 4ead8823f the code can use UIPAC masses internally.

Reply:
Changed. I just copied everything over from the other test example to make sure the way of reading in the Hessian does not influence the results. I'm planning to work through all those tests anyway, to get rid off the symlinks/outdated comments/etc.